README.md

```diff
@@ -22,12 +22,10 @@ docker run -i -t \
   -e LOG_LEVEL='debug' \
   -e LOG_FORMAT='simple' \
   -e RPC_URL='<rpc url>' \
-  -e KAFKA_SSL='true' \
-  -e KAFKA_SASL_MECHANISM='plain' \
-  -e KAFKA_USERNAME='<kafka user>' \
-  -e KAFKA_PASSWORD='<kafka password>' \
-  -e KAFKA_BROKER_ADDRESS_1='<kafka address>' \
-  -e KAFKA_TOPIC=defender \
+  -e RABBITMQ_URL='<rabbitmq url that supports ws>' \
+  -e RABBITMQ_LOGIN='<rabbitmq username>' \
+  -e RABBITMQ_PASSCODE='<rabbitmq passcode>' \
+  -e BROKER_TOPIC=defender \
```

Comment:
Maybe use `RABBITMQ_TOPIC` to have uniform env vars names?

Reply:
I've unified RABBITMQ_TOPIC and KAFKA_TOPIC into BROKER_TOPIC
(if rabbit won't work correctly we can always switch to kafka)